config/functions/functions_usage_summary.md

```diff
@@ -0,0 +1,548 @@
+The FADOUT_MS controls how long it takes the wave to  
+fade out. The WAVE_SIZE controls the width of the wave.  
+The WAVE_MS parameter controls the speed of the waves.  
+EFFECT can be used to trigger this effect by something  
+other than a blast effect.  
+
+Usage: BlastFadeoutF<FADEOUT_MS, EFFECT>  
+FADEOUT_MS: a number (defaults to 250)  
+EFFECT: a BladeEffectType (defaults to EFFECT_BLAST)  
+return value: FUNCTION  
+NOrmally returns 0, but returns up to 32768 when the  
+selected effect occurs. Then if fades back to zero over  
+FADEOUT_MS milliseconds.  
+
+Usage: OriginalBlastF<EFFECT>  
+EFFECT: a BladeEffectType (defaults to EFFECT_BLAST)  
+return value: FUNCTION  
+Original blast function. Normally returns zero, but  
+returns up to 32768 when the selected effect occurs.  
+
+## [BlinkingF](https://github.com/profezzorn/ProffieOS/blob/master/functions/blinking.h)  
+Usage: BlinkingF<A, B, BLINK_MILLIS_FUNC, BLINK_PROMILLE_FUNC>  
+BLINK_MILLIS: a number  
+BLINK_PROMILLE: a number, defaults to 500  
+BLINK_MILLIS_FUNC: FUNCTION  
+BLINK_PROMILLE_FUNC: FUNCTION  
+return value: FUNCTION  
+Switches between 0 and 32768  
+A full cycle from 0 to 328768 and back again takes BLINK_MILLIS milliseconds.  
+If BLINK_PROMILLE is 500, we select A for the first half and B for the  
+second half. If BLINK_PROMILLE is smaller, we get less A and more B.  
+If BLINK_PROMILLE is 0, we get all 0.  
+If BLINK_PROMILLE is 1000 we get all 32768.  
+
+
+## [BrownNoiseF, SlowNoise](https://github.com/profezzorn/ProffieOS/blob/master/functions/brown_noise.h)  
+Usage: BrownNoiseF<GRADE>  
+return value: FUNCTION  
+Returns a value between 0 and 32768 with nearby pixels being similar.  
+GRADE controls how similar nearby pixels are.  
+
+Usage: SlowNoise<SPEED>  
+return value: FUNCTION  
+Returns a value between 0 and 32768 which changes randomly up and  
+down over time. All pixels gets the same value.  
+SPEED controls how quickly the value changes.  
+
+## [Bump, HumpFlickerFX](https://github.com/profezzorn/ProffieOS/blob/master/functions/bump.h)  
+Usage: Bump<BUMP_POSITION, BUMP_WIDTH_FRACTION>  
+Returns different values for each LED, forming a bump shape.  
+If BUMP_POSITION is 0, bump will be at the hilt.  
+If BUMP_POSITION is 32768, the bump will be at the tip.  
+If BUMP_WIDTH_FRACTION is 1, bump will be extremely narrow.  
+If BUMP_WIDTH_FRACTION is 32768, it will fill up most/all of the blade.  
+BUMP_POSITION, BUMP_WIDTH_FRACTION: INTEGER  
+
+Usage: HumpFlickerFX<FUNCTION>  
+or: HumpFlickerF<N>  
+FUNCTION: FUNCTION  
+N: NUMBER  
+return value: INTEGER  
+Creates hump shapes that randomize over the blade.  
+The returned INTEGER is the size of the humps.  
+Large values can give the blade a shimmering look,   
+while small values look more like speckles.  
+
+## [CenterDistF](https://github.com/profezzorn/ProffieOS/blob/master/functions/center_dist.h)  
+Usage: Remap<CenterDistF<CENTER>,COLOR>  
```

Comment:
The preview is treating some of these things as HTML.
<center> disappears and becomes a newline, which is not what we want.
Either we have to quote all the < > characters (replace with &lt; &gt;) or put all the examples within backticks.

Reply:
lol It seems your reply was affected as well. I assume "CENTER" disappears

Reply:
escaping like `\<` worked.